operator/validator/event_handler.go

```diff
@@ -59,7 +58,7 @@ func (c *controller) handleOperatorRegistrationEvent(event abiparser.OperatorReg
 	}
 
 	logFields := make([]zap.Field, 0)
-	if strings.EqualFold(eventOperatorPubKey, c.operatorPubKey) || c.validatorOptions.FullNode {
+	if strings.EqualFold(eventOperatorPubKey, c.operatorPubKey) || c.validatorOptions.Mode == validator.ModeRW {
```

Comment:
Validator mode (R - non committee, RW - in committee) is a different notion than `fullnode` which is a flag in the node level that adds persistency of the entire decided history of all validators (explorer), and applies on all validators.